package.json

```diff
@@ -30,9 +30,13 @@
     "unit": "uvu test .test.js"
   },
   "dependencies": {
+    "detect-indent": "^6.1.0",
```

Comment:
It is not an option. Please improve code of our local indent detection.

Reply:
Why not? Why re-invent the wheel?... This would've prevented this bug and likely many others as well.

Reply:
Because we need this feature for only old npm version and this kib covers many unnecessary for us use cases.

Bad dependency policy is a big problem of modern JS community and I try to force more accurate policy working with them.